courses/free-courses-en.md

```diff
@@ -730,6 +730,7 @@
 * [Asynchronous Programming: The End of The Loop](https://egghead.io/courses/asynchronous-programming-the-end-of-the-loop) - Jafar Husain
 * [Beginner's Series to: JavaScript](https://youtube.com/playlist?list=PLlrxD0HtieHhW0NCG7M536uHGOtJ95Ut2) - Microsoft Developer (YouTube)
 * [Bento JavaScript Learning Track](https://bento.io/topic/javascript) (Bento)
+* [Creative Coding for Complete Beginners](https://www.youtube.com/watch?v=VpIFZ0xcJWM&list=PLUG_f-krxzVrRCOjGFwOuYj3QarVfPWXK) - Algorithmic Art(YOUTUBE)
```

Comment:
Please provide the link to the playlist directly instead of the video. also remove youtube attribution